### PR TITLE
chore: fetch-metrics.sh -> summary.sh

### PR DESCRIPTION
**Pull request by id**

### CODE REVIEW

**Review by thalesmg — APPROVED** [review]

File: scripts/summary.sh [hunk 1]
```diff
@@ -35,49 +35,64 @@ emqx_messages_input_period_second=$(curl -gs "$PROMETHEUS_URL/api/v1/query?query
...
-connections:                   $emqx_connections
-messages_received:             $emqx_messages_received
-messages_sent:                 $emqx_messages_sent
-messages_acked:                $emqx_messages_acked
-messages_publish:              $emqx_messages_publish
-messages_delivered:            $emqx_messages_delivered
-messages_dropped:              $emqx_messages_dropped
+## EMQX /monitor_current
+
+Metric                        Value
```
[the remark this answers is not in the document]
Comment (id) [reply scripts/summary.sh:49]: pandoc handles this just fine when converting to HTML, so I left it like this to not complicate further

[the remark this answers is not in the document]
Comment (id) [reply scripts/summary.sh:49]: I mean, `pandoc -f markdown -t html -o summary.html summary.md` creates valid tables

[the remark this answers is not in the document]
Comment (id) [reply scripts/summary.sh:49]: But github does not :(

**Comment by id** [comment]
Added example, copypasted as is, no extra formatting.

**Pull request merged**